ecs.php

```diff
@@ -2,29 +2,34 @@
 
 use PhpCsFixer\Fixer\ClassNotation\VisibilityRequiredFixer;
 use PhpCsFixer\Fixer\Comment\HeaderCommentFixer;
-use PhpCsFixer\Fixer\Phpdoc\PhpdocTagTypeFixer;
 use SlevomatCodingStandard\Sniffs\Commenting\InlineDocCommentDeclarationSniff;
-use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
-use Symplify\EasyCodingStandard\ValueObject\Option;
+use Symplify\EasyCodingStandard\Config\ECSConfig;
 
-return static function (ContainerConfigurator $containerConfigurator): void
-{
-    $containerConfigurator->import('vendor/sylius-labs/coding-standard/ecs.php');
+return static function (ECSConfig $config): void {
+    $config->import('vendor/sylius-labs/coding-standard/ecs.php');
 
-    $containerConfigurator->services()->set(HeaderCommentFixer::class)->call('configure', [[
-        'location' => 'after_open',
-        'header' =>
-            'This file is part of the Sylius package.
+    $config->parallel();
+    $config->paths(['src', 'spec', 'tests']);
 
-(c) Paweł Jędrzejewski
-
-For the full copyright and license information, please view the LICENSE
-file that was distributed with this source code.',
-    ]]);
-
-    $containerConfigurator->parameters()->set(Option::SKIP, [
+    $config->skip([
         InlineDocCommentDeclarationSniff::class . '.MissingVariable',
         VisibilityRequiredFixer::class => ['*Spec.php'],
         '**/var/*',
     ]);
+
+    $config->ruleWithConfiguration(
+        HeaderCommentFixer::class,
+        [
+            'location' => 'after_open',
+            'comment_type' => HeaderCommentFixer::HEADER_COMMENT,
+            'header' => <<<TEXT
+This file is part of the Sylius package.
+
+(c) Paweł Jędrzejewski
```

Comment:
Should it use new `(c)` instead?

Reply:
Yeah, I didn't want to put too many changes in this PR, but since I'm already at it, I might as well update everything 🤷